amplipi/ctrl.py

```diff
@@ -541,9 +547,39 @@ def _sync_stream_info(self) -> None:
       streams.append(stream)
     self.status.streams = streams
 
+  def _update_serial(self) -> None:
+    expanders = []
+    # Preamp Boards
+    eeprom_preamp = EEPROM(BoardType.PREAMP)
+    eeprom_streamer = EEPROM(BoardType.STREAMER_SUPPORT)
+    if eeprom_preamp.present():
+      bi = eeprom_preamp.read_board_info()
+      if bi is not None:
+        self._serial = int(bi.serial)
+        for board in [0x10, 0x18, 0x20, 0x28, 0x30]:
```

Comment:
I would recommend
```suggestion
        for board in rt._DEV_ADDRS[1:]: # Check for expanders
```
Although maybe eeprom.py should be handling the unit number -> I2C address conversion...

And either way I guess _DEV_ADDRS shouldn't be marked as private anymore, or moved to a getter.